### PR TITLE
Initial commit: update pip3-skeleton with project specifics

### PR DESCRIPTION
**Pull request by DiamondJoseph**
- Limit python compatibility to those versions supported by BlueAPI

### CODE REVIEW

**Comment by DiamondJoseph** [comment]
Tagged @gilesknap for review to make sure I have removed all the correct skeleton things and @keithralphs as he understands the purpose of the repository and can check if my docs make sense.

File: .github/workflows/code.yml [hunk 1]
```diff
@@ -32,13 +32,8 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-latest"] # can add windows-latest, macos-latest
-        python: ["3.8", "3.9", "3.10", "3.11"]
-        install: ["-e .[dev]"]
-        # Make one version be non-editable to test both paths of version code
```
Comment (gilesknap) [inline .github/workflows/code.yml:37]: You have removed the non-editable test which does pick up a few issues that editable tests don't.

To resolve this remove "3.9" from your matrix above and then add:
```python
      include:		
         - os: "ubuntu-latest"		
           python: "3.9"		
           install: ".[dev]"
```

**Review by gilesknap — COMMENTED** [review]
Looks good. One minor comment.

File: .github/workflows/code.yml (hunk 1 above)
Comment (DiamondJoseph) [reply .github/workflows/code.yml:37]: It's actually testing against the non-editable version of all 3 versions of Python we claim to support:

       os: ["ubuntu-latest"] # can add windows-latest, macos-latest
        python: ["3.9", "3.10", "3.11"]
        install: [".[dev]", "-e .[dev]"]

Runs against every combinationf of python, install[, os]
image

**Review by gilesknap — APPROVED** [review]

**Pull request merged**